e2e/awscluster.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
```

Comment:
The files on these folder are meant to be used as a quickly manual test from the Tilt UI.